TreeList: add Overview (#6633)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeList/02 Overview/05 Elements.md

> Below is a map of elements. Click on an element to navigate to a topic related to that element.

Reviewer: ```suggestion
Click an element in the map below to navigate to the topic related to that element.
```
